CHANGELOG.asciidoc

```diff
@@ -0,0 +1,501 @@
+ifdef::env-github[]
+NOTE: Release notes are best read in our documentation at
+https://www.elastic.co/guide/en/apm/agent/go/current/release-notes.html[elastic.co]
+endif::[]
+
+////
+All notable changes to this project will be documented in this file.
+See Conventional Commits (https://conventionalcommits.org) for commit guidelines.
+////
+
+////
+[[release-notes-x.x.x]]
+==== x.x.x (YYYY/MM/DD)
+
+[float]
+===== Breaking changes
+
+[float]
+===== Features
+* Cool new feature: {pull}2526[#2526] {commit}d4f181f[#d4f181f]
+
+[float]
```

Comment:
have to include `Performance Improvements` to the list. example - https://github.com/elastic/apm-agent-rum-js/blob/master/CHANGELOG.md#performance-improvements